exchange/utils_test.go

```diff
@@ -55,6 +55,370 @@ func assertReq(t *testing.T, bidderRequests []BidderRequest,
 	}
 }
 
+func TestSplitImps(t *testing.T) {
+	testCases := []struct {
+		description   string
+		givenImps     []openrtb.Imp
+		expectedImps  map[string][]openrtb.Imp
+		expectedError string
+	}{
+		{
+			description:   "Nil",
+			givenImps:     []openrtb.Imp{},
+			expectedImps:  map[string][]openrtb.Imp{},
```

Comment:
This test case seems to be the same as the next one. Did you mean to assign `nil` value to `givenImps` here?

Reply:
Yes, I did. Thank you. Fixed.